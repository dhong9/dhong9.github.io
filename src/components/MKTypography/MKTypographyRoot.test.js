import renderer from "react-test-renderer";
import MKTypographyRoot from "components/MKTypography/MKTypographyRoot";

// Material Kit 2 React themes
import { ThemeProvider } from '@mui/material/styles';
import theme from "assets/theme";

describe("MKTypographyRoot", () => {
    it("renders", () => {
        const component = renderer.create(
            <ThemeProvider theme={theme}>
                <MKTypographyRoot
                ownerState={{
                    color: "text",
                    textTransform: "capitalize",
                    verticalAlign: "middle",
                    fontWeight: "bold",
                    opacity: 1,
                    textGradient: false,
                }}
                >
                    Test Typography
                </MKTypographyRoot>
            </ThemeProvider>
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});