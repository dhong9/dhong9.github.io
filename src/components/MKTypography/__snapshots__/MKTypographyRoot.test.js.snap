// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MKTypographyRoot renders 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 css-4qj1a3-MuiTypography-root"
>
  Test Typography
</p>
`;
